tnt-integration-tests/src/main/java/org/dcsa/tnt/itests/v2/GetEventsIT.java

```diff
@@ -98,15 +97,18 @@ public void getTransportEvents() {
         .jsonPath()
         .getList(".", EventTO.class);
 
-    events.forEach(e -> assertInstanceOf(TransportEventTO.class, e));
+    events.forEach(e -> assertInstanceOf(TransportEventPayloadTO.class, e.payload()));
   }
 
+  /*
+  TODO rewrite/update these tests for 3.0 spec filters
```

Comment:
Created https://dcsa.atlassian.net/browse/DDT-1265 to update filters and integration tests